src/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {FC, ChangeEvent} from 'react';
+import styled from 'styled-components';
+
+import {Box} from '../Box';
+
+import {theme} from '../theme';
+
+interface IFakeInput {
+  checked?: boolean;
+}
+
+type RadioButtonProps = {
+  id: string;
+  label: string;
+  checked?: boolean;
+  value: string;
+  type?: string;
```

Comment:
since this component is precisely a `RadioButton`, is it true to say that the type would always be "**radio**" and hence removed as a prop?

Reply:
Agreed!